.gitignore

```diff
@@ -20,7 +20,6 @@ release/
 !/logs/.htaccess
 
 /vendor/
-composer.lock
```

Comment:
We should always commit the lock files. Same reason why we should commit `package-lock.json`